resources/model/OpenStudio.idd

```diff
@@ -25376,10 +25376,25 @@ OS:LoadProfile:Plant,
        \type real
        \units m3/s
        \ip-units gal/min
-  A6 ; \field Flow Rate Fraction Schedule Name
+  A6 , \field Flow Rate Fraction Schedule Name
        \required-field
        \type object-list
        \object-list ScheduleNames
+  A7 , \field Plant Loop Fluid Type
+       \type choice
+       \key Water
+       \key Steam
+       \required-field
+  N2 , \field Degree of SubCooling
+       \note This field is used only when Plant Loop Fluid Type=Steam.
+       \units C
+       \minimum 1.0
+       \required-field
+  N3 ; \field Degree of Loop SubCooling
+       \note This field is used only when Plant Loop Fluid Type=Steam.
+       \units C
+       \minimum 10.0
+       \required-field
```

Comment:
have defaults in E+, so make them required-field and set in Ctor. I like where this is going :+1: